app.py

```diff
@@ -214,7 +214,7 @@ def setDemoMode(val: bool = False):
         case _:
             python_repl = os.path.join(venv_dir, "bin", "python3")
 
-driver_scripts = find_driver_scripts(temp_dir_path)
+driver_scripts = []
 app.starter_script = st.selectbox("Training Script:", driver_scripts)
 training_cmd: List[str] = None
```

Comment:
this line should be = []

Reply:
which line sir?

Reply:
line no 217 shouldn't be empty. U can see right there that in 218 its referenced, u ill get empty list for driver scripts hence start_script is going to be None